pkg/ratelimit/option.go

```diff
@@ -78,7 +78,11 @@ func UpdateDimensionConfig(cfg *DimensionConfig) Option {
 			return InAllowList
 		}
 		lim, _ := l.limiters.LoadOrStore(label, newLimiter())
-		return lim.(*limiter).updateDimensionConfig(cfg)
+		status := lim.(*limiter).updateDimensionConfig(cfg)
+		if status&QPSDeleted != 0 && status&ConcurrencyDeleted != 0 {
+			l.limiters.Delete(label)
```

Comment:
What will happen if this is not removed?

Reply:
leave a config in output that is meaningless